src/assets/styles/_mixins.scss

```diff
@@ -0,0 +1,13 @@
+// Mixins
+
+@mixin borderShadow() {
+  box-shadow: 0 1px 0 var(--calcite-app-border);
+}
+
+@mixin lineClamp($lines: 2) {
```

Comment:
Suggest adding a comment with a link where this technique is described.

Reply:
comment added

Reply:
ty sir